Source/tusdotnet/tusdotnet.csproj

```diff
@@ -5,7 +5,7 @@
     <Copyright>Copyright Stefan Matsson 2016</Copyright>
     <AssemblyTitle>tusdotnet</AssemblyTitle>
     <Authors>Stefan Matsson</Authors>
-    <TargetFrameworks>net452;netstandard1.3</TargetFrameworks>
+    <TargetFrameworks>net452;netstandard1.3;netcoreapp3.0</TargetFrameworks>
```

Comment:
Could probably change this to netstandard2.1 to also include .NET Core 3.1 and .NET 5. This would reduce package size and allow us not to have to update the target frameworks on each new framework release (e.g. .NET6, 7 etc). Minor issue :)

Reply:
Unfortunately, we need to target netcoreapp3.0 because there isn't a 3.0 version of `Microsoft.AspNetCore.Http.Abstractions` on nuget (https://github.com/dotnet/aspnetcore/issues/18408). As far as i know, .NET 5, 6, 7 etc. can still use all packages targeting netcoreapp3.0.

Reply:
That's OK. 👍 Currently working on adding the checksum-trailer extension to tusdotnet and will have to add a build target for netcoreapp3.1 anyway. Thanks for the info!